core/src/test/java/com/linecorp/armeria/server/logging/LoggingServiceTest.java

```diff
@@ -154,6 +155,165 @@ public void infoLevel() throws Exception {
                             ", trailers: " + RESPONSE_TRAILERS);
     }
 
+    @Test
+    public void  map_request_level_test_header_as_warn() throws Exception {
```

Comment:
- double spaces between `void` and the method name.
- Could you use camel-case notation?